docs/src/guides/functional/versioning.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Versioning Entities
+navTitle: Versioning
+description: Learn how to work with entity versions.
+order: 100
+group: 3-functional-api
+---
+
+## Hub Entity Versioning
+
+The versioning feature allows developers to create multiple versions of any `Item`-backed entity. Versions are stored as item resources.
+
+### Searching for versions of an entity
+
+The [`searchVersions`](/hub.js/api/common/searchVersions) function can be used to get all the versions of an entity. It takes an item id and an [`IHubUserRequestOptions`](/hub.js/api/common/IHubUserRequestOptions) and returns an array of ['IVersonMetadata](/hub.js/api/common/IVersionMetadata).
+
+### Getting and working with a particular version of an entity
+
+The [`getVersion`](/hub.js/api/common/getVersion) function can be used to get a particular version of an entity. It takes an item id, a version id, and and an [`IHubUserRequestOptions`](/hub.js/api/common/IHubUserRequestOptions) and returns an ['IVerson](/hub.js/api/common/IVersion).
+
+Once you have an entity and a version, you can apply the version to the entity using the [`applyVersion`](/hub.js/api/common/applyVersion) function. It takes an ['IModel'](/hub.js/api/common/IHubUserRequestOptions), an ['IVerson](/hub.js/api/common/IVersion), and an optional `includeList` and returns an IModel with the versioned data applied.
```

Comment:
I think this is pretty pragmatic, but I wonder if we should tie this into the `IHub<Entity>` layer vs the `IModel` layer? I know that to work with the current Site, we need to support `IModel`, but if we make that the _main_ way, we'll cause confusion down the line. Maybe this section should be "Working with Site Layout Versions Today", and have another section "Working with Hub Entity Versions" below, and can be "Coming Soon", and then we can design the integration so it's seamless at the Entity layer (which is what Hub Components and App-v-next should deal with vs `IModel`)

Reply:
aaaand I read further down, and now I see that it is tied into the classes, so maybe this guide should focus on the "new" workflows we want to encourage, and have a section at the end on using this for Sites in Q1/2023